### PR TITLE
Add equations-1.2 packages

### PR DESCRIPTION
**Pull request by mattam82**

### CODE REVIEW

**Comment by palmskog** [comment]
Can you please convert these packages to OPAM 2.0 format? Also, please delete the unnecessary `remove` line and add `tags` with metadata such as logical path prefix (`logpath`).

**Comment by mattam82** [comment]
@palmskog sure, done now.

File: released/packages/coq-equations/coq-equations.1.2+8.8/opam [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+license: "LGPL 2.1"
+synopsis: "A function definition package for Coq"
+description: """
+Equations is a function definition plugin for Coq, that allows the
+definition of functions by dependent pattern-matching and well-founded,
+mutual or nested structural recursion and compiles them into core
+terms. It automatically derives the clauses equations, the graph of the
+function and its associated elimination principle.
+"""
+flags: light-uninstall
```
Comment (palmskog) [inline released/packages/coq-equations/coq-equations.1.2+8.8/opam]: Thanks for the conversion, but now the package fails linting due to this line - please remove it (and also for other package).

Comment (mattam82) [reply released/packages/coq-equations/coq-equations.1.2+8.8/opam]: Right, I blindly copied from `coq-bignums.8.9.0` which had it

**Comment by palmskog** [comment]
Everything looks good now, I'll merge as soon as CI passes.

**Pull request merged**